media-video/ffmpeg/ffmpeg-9999.ebuild

```diff
@@ -298,6 +298,7 @@ src_prepare() {
 		export revision=git-N-${FFMPEG_REVISION}
 	fi
 	default
+	cat "${FILESDIR}"/libffmpeg.mak >> Makefile || die
```

Comment:
Please use the patch you sent upstream

Reply:
just to be clear: please use a patch instead of cat; the one you sent upstream or an improved version of it is even better

also, please include the link to the upstream ml thread in the patch headers: this will likely be carried over for years so the more information you can put there the better